build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = 'cloud.eppo'
-version = '3.1.0-SNAPSHOT'
+version = '3.2.0-SNAPSHOT'
```

Comment:
API modifications:
- `loadConfigurationAsync` method on `BaseEppoClient`
- `Configuration` moved to `api` package.